java/gae-flexible-helloworld/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<%@ page import="com.example.bigtable.BigtableHelloWorld" %>
+<!-- [START_EXCLUDE] -->
+<%--
+  ~ Copyright (c) 2016 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you
+  ~ may not use this file except in compliance with the License. You may
+  ~ obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+  ~ implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  --%>
+<!-- [END_EXCLUDE] -->
+<html>
+<head>
+  <link href='//fonts.googleapis.com/css?family=Marmelad' rel='stylesheet' type='text/css'>
+  <title>Hello App Engine Flexible</title>
+</head>
+<body>
+    <h1>Hello App Engine -- Flexible!</h1>
+
+  <p>This is <%= BigtableHelloWorld.doHelloWorld() %>.</p>
```

Comment:
Closing `</p>` tags are optional and [optional tags should be left out according to the Google HTML style guide](https://google.github.io/styleguide/htmlcssguide.xml?showone=Optional_Tags#Optional_Tags)